testing/test_aux.out

```diff
@@ -101,16 +103,18 @@ port-02 module_config modules
     "enabled": true
   },
   "hold": {
-    "enabled": false
+    "enabled": true 
   },
   "macoui": {
     "enabled": true
   },
   "nmap": {
-    "enabled": true
+    "enabled": true,
+    "timeout": 60
```

Comment:
was there a case somewhre that was testing that the JSON paramater was actually doing something. I'm not sure what is currently being tested with the hold timeout, but it might be the default (350?).   Not sure how to easily verify that it works, but would like to at least be able to tell from the logs.